### PR TITLE
Test for react three fiber component

### PR DESCRIPTION
**Pull request by Angatupyry**
## What's new

- Test added for react three fiber components
## Self-checks

- [ ] I have prototyped this new feature (if necessary) on Figma 
- [ ] I'm familiar with and follow this [Typescript guideline](https://basarat.gitbook.io/typescript/styleguide)
- [ ] I added unit-tests for new components
- [ ] I tried testing edge cases
- [ ] I tested the behavior of the components that interact with the backend, with an e2e test

### CODE REVIEW

File: packages/react-components/lib/map/three-fiber/circle-shape.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { CircleShape } from './circle-shape';
+import { Euler, Vector3 } from 'three';
+import { makeRobotData } from '../test-utils.spec';
+import { Canvas } from '@react-three/fiber';
+
+describe('CircleShape', () => {
+  it('debería renderizar un círculo y una línea correctamente', () => {
```
Comment (aaronchongth) [inline packages/react-components/lib/map/three-fiber/circle-shape.spec.tsx:9]: We should probably keep descriptions in english

File: packages/react-components/lib/map/three-fiber/robot-three-maker.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
-import { Circle, Line, Text } from '@react-three/drei';
 import { ThreeEvent } from '@react-three/fiber';
 import React from 'react';
 import { Euler, Vector3 } from 'three';
+import { CircleShape } from '.';
```
Comment (aaronchongth) [inline packages/react-components/lib/map/three-fiber/robot-three-maker.tsx:4]: Typescript should be able to handle this, but we should try not to import in a circular fashion anyway (robot-three-maker->index->robot-three-maker)

**Review by aaronchongth — CHANGES_REQUESTED** [review]
Looks great! I've some basic comments, and after resolving the merge conflict, we will be good to go!

File: packages/react-components/lib/map/three-fiber/circle-shape.spec.tsx (hunk 1 above)
Comment (Angatupyry) [reply packages/react-components/lib/map/three-fiber/circle-shape.spec.tsx]: Thanks for catching it!
Done cc14bed4b0ee59f55fc6efe1f29c520a1eee9028!

File: packages/react-components/lib/map/three-fiber/robot-three-maker.tsx (hunk 2 above)
Comment (Angatupyry) [reply packages/react-components/lib/map/three-fiber/robot-three-maker.tsx]: Done 9e45765c0b014774b72ef8908b5316d612c220cf

**Review by aaronchongth — APPROVED** [review]
Thanks for handling this! LGTM

**Pull request merged**